docs/debug-kernel.md

```diff
@@ -0,0 +1,45 @@
+# Get Clear Containers Kernel logs
+
+Clear Containers runtime launches a special virtual machine when a new pod is
+created. This VM uses a custom kernel inside the VM.
+
+The kernel boot logs are disabled by default. However these logs may be desired
+when the kernel is customized by the end user, or if the virtual machine boot
+fails due to the kernel.
+
+To provide a debug log of the kernel boot, Clear Containers supports an option
+to enable kernel boot logs using the configuration file.
+
+1- Enable debug in configuration.toml
+
+```
+enable_debug = true
+```
+
+2- Run a container to generate the logs
+
+```
+sudo docker run -ti busybox true
```

Comment:
For consistency with the other docs, can you add a `$ ` shell prompt prefix for all the command-lines?

Reply:
done. even though not having the $ makes cut and paste easy :)